source/_integrations/zwave_js.markdown

```diff
@@ -139,11 +153,61 @@ Value Notification example:
 }
 ```
 
+## Current Limitations
+
+As this integration is still in the early stages there are some important limitations to be aware of.
+
+- Some advanced CommandClasses are not yet fully implemented in Z-Wave JS. No worries as all the most common devices are already working. You can track status [here](https://github.com/zwave-js/node-zwave-js/issues/6). Actually the only one that impacts HA users is Barrier Operator CC (= garage door controllers) but the good news is that this is [almost finished](https://github.com/zwave-js/node-zwave-js/pull/1337)!
```

Comment:
> Actually the only one that impacts HA users is Barrier Operator CC

I don't think this is a fair assumption. As an example, I saw some comments about thermostat being unusable without the fan mode CC being available.

Reply:
I've adjusted the text